README.md

```diff
@@ -122,6 +122,8 @@ To compile all the lessons in your terminal, you can
 or
 2. navigate to the project directory using Windows PowerShell and run `.\win-compile-all.ps1`
 
+Additionally, if you are able to install WSL (Windows Subsytem for Linux) onto your Windows machine, you can follow the mac/linux instructions more directly. 
```

Comment:
This should be the preferred method for windows now. Have you tested it out?

Reply:
Yes I have! Works well, but should not be the default since it does not work on all windows machines (requires virtualization).

Reply:
@drassaby Please approve if all good!

Reply:
Ok! Do you think it should still be the recommended path (when available) though? We could keep the other instructions around for Windows troubleshooting

Reply:
@John-HarringtonNZ happy to approve either way! But wanna make sure you see this question first

Reply:
Honestly probably not since it requires updating to a slightly unstable Windows version at this point (hopefully not in the future though).